sass/base/_structure.scss

```diff
@@ -71,7 +71,7 @@ body {
 }
 
 .container-fluid {
-    @include legacy-pie-clearfix;
+    @include pie-clearfix;
```

Comment:
I'm not familiar with this, do you know what it is doing? What's the difference between `pie-clearfix` vs `clearfix`?

Reply:
n/m, found! :)